lib/directive.js

```diff
@@ -134,6 +141,12 @@ function init(datetime, $log, $document){
 			}
 		}
 
+		if (angular.isDefined(attrs.datetimeTimezone)) {
+			if (/[+-]\d{4}/.test(attrs.datetimeTimezone)) {
```

Comment:
@eight04 Maybe need to add additional check for ```/[+-]\d{2}:\d{2}/``` pattern here?

Reply:
You're right. Since `parser.setTimezone` accepts colon, the directive should accept it as well.